onnxruntime/hosting/http/util.cc

```diff
@@ -52,5 +53,31 @@ boost::beast::http::status GetHttpStatusCode(const protobufutil::Status& status)
   }
 }
 
+SupportedContentType GetRequestContentType(const HttpContext& context) {
+  if (context.request.find("Content-Type") != context.request.end()) {
+    if (context.request["Content-Type"] == "application/json") {
+      return SupportedContentType::Json;
+    } else if (context.request["Content-Type"] == "application/octet-stream") {
```

Comment:
Why not make this more specific? Like [`application/x-protobuf`](https://developers.google.com/api-client-library/java/google-http-java-client/reference/1.20.0/com/google/api/client/protobuf/ProtocolBuffers).

Reply:
Since AzureML monitoring system has not been decided yet. I did not put specific MIME type here. But I'd like to accept more types to make the client easier.